docs/Release_Notes.md

```diff
@@ -53,7 +51,7 @@ cd ../ecf
 VERSION FILE CHANGES
 --------------------
 
-* `versions/run.ver` - change `version=v16.3.20`, `gfs_ver=v16.3.20`, and `obsproc_ver=v1.3`
+* `versions/run.ver` - change `version=v16.3.21`, `gfs_ver=v16.3.21`, and `obsproc_ver=v1.3`
```

Comment:
```suggestion
* `versions/run.ver` - change `version=v16.3.21` and `gfs_ver=v16.3.21`
```

Reply:
@KateFriedman-NOAA Just made the small edits.

Reply:
Thanks @YaliMao-NOAA !